sycl/include/sycl/ext/intel/esimd/detail/memory_intrin.hpp

```diff
@@ -931,7 +923,7 @@ __esimd_media_ld(TACC handle, unsigned x, unsigned y)
   assert((handle != __ESIMD_NS::detail::SLM_BTI) &&
          "__esimd_media_ld cannot access SLM");
 
-  sycl::detail::getESIMDDeviceInterface()->sycl_get_cm_image_params_index_ptr(
+  sycl::detail::getESIMDDeviceInterface()->sycl_get_cm_image_params_ptr(
       handle, &readBase, &imgWidth, &imgHeight, &bpp, &mutexLock);
```

Comment:
So many params, can't it be just a struct and passing a reference to it?

Reply:
The purpose of this function (and `sycl_get_cm_buffer_params_ptr`) is to get buffer/image information contained in `_pi_image` and `_pi_buffer` without dependency on their definitions. I think passing parameters only with common data type is appropriate here - rather than adding another `struct` definition.